.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Bug report
+about: Create a report to help us improve
+labels: bug
+---
+
+## Guidelines
+
+Please note that GitHub issues are only meant for bug reports/feature requests. If you have questions on how to use the Neo4j Connector for Apache Spark,
+please ask on [the Neo4j Discussion Forum](https://community.neo4j.com/c/integrations/18) instead of creating an issue here.
+
```

Comment:
please add a note here where we ask for the following information:
* the code of the spark job
* the structure of the Dataframe
* did you define the constraints/indexes?
* if you're you using any Spark Cloud provider please specify it